LondonBritishLibrary/orient/BLorient2198.xml

```diff
@@ -30,27 +30,136 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msIdentifier>
                         <repository ref="INS00001BL"/>
                         <idno>BL Oriental 2198</idno>
+                        <altIdentifier>
+                            <idno>Strelcyn 15</idno>
+                        </altIdentifier>
                     </msIdentifier>
                     
                     <msContents>
                         <summary/>
                         <msItem xml:id="ms_i1">
                             <locus from="1r" to="13r"/>
                             <title type="incomplete" ref="LIT3585Taamme"/>
-                            <note>The beginning of the text is missing.</note>
-                            <textLang mainLang="gez"></textLang>
+                            <note>Numbered from second to seventh miracle. The miracle numbered first and the beginning of the second miracle is 
+                                missing.</note>
+                                <msItem xml:id="ms_i1.1">
+                                    <locus from="1r" to="3r"/>
+                                    <title type="incomplete">Second miracle</title>
+                                    <note>How the Virgin Mary accepted the Book of her Miracles compiled by St Hildefonsus, Bishop of Toledo. 
+                                        (Cf. <bibl><ptr target="bm:Budge1900Miracles"/><citedRange unit="page">20-24</citedRange></bibl>
+                                        <bibl><ptr target="bm:Cerulli1943Maria"/><citedRange>311-312</citedRange></bibl>).</note>
+                                    <incipit xml:lang="gez">ወበከመ፡ <gap reason="lost"/> ገብርከ፡ በዓለ፡ ዜናየ፡ ዘዜነወሂ፡ መልአክ፡ ወአስተፈሣሕከ፡ ኵሎ፡ ሰብእ፤ በእንቲአየ፡ 
+                                        ወአንሰ፡ እዌስክ፡ <gap reason="ellipsis"/> ወሶበ፡ ተፈጸመ፡ መዋዕለ፡ ሕይወቱ፡ ወገብአ፡ ኀበ፡ ምሕረተ፡ እግዚአብሔር፡ ርእይዋ፡ ለይእቲ፡ ልብስ፡ ውስተ፡ 
+                                        መዛግብተ፡ ቤተ፡ ክርስቲያን፡ እንዘ፡ ንጽሕት፡ ፈድፋደ። ወሶበ፡ ተሰይመ፡ እምድኅሬሁ፡ ኤጲስ፡ ቆጶስ፤ ፈቀደ፡ ይልበሳ፡ ለይእቲ፡ ልብስ፡ ወይንበር፡ ዲበ፡ መንበሩ፡ 
+                                        ለኤጲስ፡ ቆጶስ፤ ዳቅስዮስ። </incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2">
+                                    <locus from="3v" to="5v"/>
+                                    <title>Third miracle</title>
```

Comment:
NB: for the Miracles of Mary you can also use PEMM's incipit tool, it is faster than Bm. 
https://pemm.princeton.edu/research/incipit-tool
just as an example, typing in the incipit you get the result Miracle 156 https://pemm.princeton.edu/stories/156, which is mapped to Beta maṣāḥǝft ID LIT3924Miracle